Cargo.toml

```diff
@@ -71,9 +71,9 @@ getrandom = "0.2.1"
 # Async
 tokio = { version = "1.0", features = ["rt-multi-thread"] }
 futures-util = "0.3.5"
-aws-config = "0.15.0"
-aws-sdk-s3 = "0.15.0"
-aws-smithy-types-convert = { version = "0.45.0", features = ["convert-chrono"] }
+aws-config = "0.46.0"
```

Comment:
lol wtf, 31 major versions in 6 months?
good for them I guess 😂

Reply:
they merged some crates and skipped versions :)